cmd/releasego/update-azure-linux.go

```diff
@@ -96,19 +107,79 @@ func updateAzureLinux(p subcmd.ParseFunc) error {
 		return err
 	}
 
-	ar := txtar.Archive{
-		Comment: []byte("Bump version to " + assets.GoVersion().Full()),
-		Files: []txtar.File{
-			{Name: cgManifestFilepath, Data: cgManifestBytes},
-			{Name: golangSignaturesFilepath, Data: golangSignaturesFileBytes},
-			{Name: golangSpecFilepath, Data: []byte(golangSpecFileContent)},
-		},
+	ref, _, err := client.Git.GetRef(ctx, owner, repo, baseBranch)
+	if err != nil {
+		return fmt.Errorf("Failed to get ref: %v", err)
+	}
+
+	newRef := &github.Reference{
+		Ref:    github.String(updateBranch),
+		Object: &github.GitObject{SHA: ref.Object.SHA},
+	}
+
+	if _, _, err = client.Git.CreateRef(ctx, owner, repo, newRef); err != nil {
```

Comment:
Will this fail if the ref already exists? Maybe we should create a new ref instead? (In case of a rerun.)

Reply:
Talked about this in sync: letting it fail helps avoid potentially generating more PRs/branches than we want, and we will have a parameter on the release pipeline to let us specify a new name upon rerun if we truly want a new branch/PR. This also preserves history rather than overwrite an existing branch/PR.